spec/models/reviewable_spec.rb

```diff
@@ -244,6 +244,55 @@
     end
   end
 
+  context ".recent_list_with_pending_first" do
+    fab!(:pending_reviewable1) { Fabricate(:reviewable, score: 150) }
+    fab!(:pending_reviewable2) { Fabricate(:reviewable, score: 100) }
+    fab!(:approved_reviewable1) { Fabricate(:reviewable, score: 300) }
+    fab!(:approved_reviewable2) { Fabricate(:reviewable, score: 200) }
```

Comment:
I'm not familiar here but how does the score determine whether a reviewable is pending or approved? The scores feel very magical to me which means if the guidelines for the score is ever changed, this test will break.

Reply:
O I guess all reviewables are pending by default?

Reply:
Yeah they're pending by default. The score determines how a reviewable should be sorted and if it should be shown to mods, but it never affects the status.